### DISCUSSION TITLE
Select multipe fields for mailinglist tags

### DISCUSSION THREAD

**Discussion by WHITE-developer** [solspace/craft-freeform#489] [Feature Requests]
Hi,

We're using the mailchimp mailinglist integration and we would like to select multiple fields to the tag setting.

Is it possible to create a checkbox instead of a dropdown for the Tags selection:
![image](https://user-images.githubusercontent.com/22771556/199490051-f6657c75-5511-47ff-8257-3e808fb32f34.png)

**Comment by kjmartens** [comment]
Hi @WHITE-developer,

Unfortunately this is not currently available. However, we do have plans to improve API integrations in the future, so this might be more likely to happen then. 🙂
